lib/python/ray/worker.py

```diff
@@ -30,10 +30,10 @@
 SILENT_MODE = 3
 
 def random_object_id():
-  return photon.ObjectID("".join([chr(random.randint(0, 255)) for _ in range(20)]))
+  return photon.ObjectID(random_string())
 
 def random_string():
-  return "".join([chr(random.randint(0, 255)) for _ in range(20)])
```

Comment:
Purely aesthetic, but I think it'd be better to define `random_string` before `random_object_id` since `random_object_id` uses `random_string`.